src/lib/Preview.js

```diff
@@ -803,6 +806,15 @@ class Preview extends EventEmitter {
             this.setupUI();
         }
 
+        // Check to see if there are http interceptors and load them
+        if (this.options.responseInterceptor) {
```

Comment:
It doesn't seem like these necessarily belong in `handleTokenResponse`. Can they be added earlier, maybe in the constructor?

Reply:
We can add them early, the interceptors are added to preview during the `show()` call. I can refactor to add it on instantiation if that is something we want to do.

Reply:
I think having them in `.show` makes sense, too. It looks like they're in the mostly-unrelated `handleTokenResponse` method right now, which was my concern.

Reply:
Sounds good, I agree that handleTokenResponse isn't the best place either. I'll update accordingly